com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -34,27 +34,92 @@ modules:
         sha256: 723f24f4d65fc8d99b33a002fbbb3771d4cc9d664c97085bf37f3997ae8063af
         type: archive
 
-  - name: pikepdf
+  - name: pikepdf-deps
     buildsystem: simple
     build-commands:
-      - pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} lxml-*.whl pikepdf-*.whl Pillow-*.whl img2pdf-*.tar.gz python_dateutil-*.whl
+      - pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} lxml-*.whl Pillow-*.whl python_dateutil-*.whl
     sources:
-      - url: https://files.pythonhosted.org/packages/f2/fd/43296bd14ece0cf6c355826587cd3db9a787d19b178b3886d84dd6263fa1/pikepdf-2.4.0-cp38-cp38-manylinux2010_x86_64.whl
-        sha256: 5b05745d1c0772925929a2758898faae238ed9c7d1107f512128931355091640
-        type: file
       - url: https://files.pythonhosted.org/packages/52/a5/98a73a83bb06d271c3f915a4a13f9613fdd8a685524f46d9733eebeb55ce/lxml-4.6.2-cp38-cp38-manylinux1_x86_64.whl
+        only-arches:
+          - x86_64
         sha256: 2e5cc908fe43fe1aa299e58046ad66981131a66aea3129aac7770c37f590a644
         type: file
+      - url: https://files.pythonhosted.org/packages/f2/14/241df8983a55148e6f8b10d55dfc024ab30ca4bb2ba633d0b40ad789c8ac/lxml-4.6.2-cp38-cp38-manylinux2014_aarch64.whl
+        only-arches:
+          - aarch64
+        sha256: 50c348995b47b5a4e330362cf39fc503b4a43b14a91c34c83b955e1805c8e308
+        type: file
       - url: https://files.pythonhosted.org/packages/c9/57/a6c81c1d6b8973d288ec6df0993a71a61318c862afc1de71fd3aa1ba1dcd/Pillow-8.1.0-cp38-cp38-manylinux1_x86_64.whl
+        only-arches:
+          - x86_64
         sha256: cdbbe7dff4a677fb555a54f9bc0450f2a21a93c5ba2b44e09e54fcb72d2bd13d
         type: file
-      - url: https://files.pythonhosted.org/packages/80/ed/5167992abaf268f5a5867e974d9d36a8fa4802800898ec711f4e1942b4f5/img2pdf-0.4.0.tar.gz
-        sha256: eaee690ab8403dd1a9cb4db10afee41dd3e6c7ed63bdace02a0121f9feadb0c9
+      - url: https://files.pythonhosted.org/packages/b5/bd/323b51849a13aa16ba48701554ff5da304c7b38b1ab446fa6718513de6b4/Pillow-8.1.0-cp38-cp38-manylinux2014_aarch64.whl
+        only-arches:
+          - aarch64
+        sha256: f50e7a98b0453f39000619d845be8b06e611e56ee6e8186f7f60c3b1e2f0feae
         type: file
       - url: https://files.pythonhosted.org/packages/d4/70/d60450c3dd48ef87586924207ae8907090de0b306af2bce5d134d78615cb/python_dateutil-2.8.1-py2.py3-none-any.whl
         sha256: 75bb3f31ea686f1197762692a9ee6a7550b59fc6ca3a1f4b5d7e32fb98e2da2a
         type: file
 
+  - name: pikepdf-bin
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} pikepdf-*.whl
+    only-arches:
+      - x86_64
+    sources:
```

Comment:
I wonder if we should build x86_64 also from source. But we can leave it as is for now.

Reply:
yes we could switch to that too.

Reply:
current PR does that now.